src/content/components/search/usage.mdx

```diff
@@ -1,21 +1,18 @@
 ---
 title: Search
-tabs: ['Code', 'Usage','Style']
+tabs: ['Code', 'Usage', 'Style']
 ---
+
 ## General guidance
 
 _Search_ enables users to specify a word or a phrase to find particular relevant pieces of content without the use of navigation. Search can be used as the primary means of discovering content, or as a filter to aid the user in finding content.
 
-
-
-
 ## Variations
 
-| Search type   | Purpose                                                                                                                                                     |
-| ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| _Large search_      | This should be used at a global level, when the user is searching content within a page view.                                                                                             |
-| _Small search_  | Choose small when there are space constraints in your design. It can also be component specific. For example, small search can be used to filter data within a [data table](/components/data-table). |
-
+| Search type    | Purpose                                                                                                                                                                                              |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Large search_ | This should be used at a global level, when the user is searching content within a page view.                                                                                                        |
+| _Small search_ | Choose small when there are space constraints in your design. It can also be component specific. For example, small search can be used to filter data within a [data table](/components/data-table). |
```

Comment:
```suggestion
| Small search | Choose small when there are space constraints in your design. It can also be component-specific. For example, small search can be used to filter data within a [data table](/components/data-table). |
```